### PR TITLE
Feature/themed logo

### PR DESCRIPTION
**Pull request by petrKavulok**
Preparation work for dark themed logo. Renaming logo files.

TODO:
- [x]  asab-webui update when https://github.com/TeskaLabs/asab-webui/pull/375 is finished
- [x] Changelog updates 

### ASAB-WEBUI submodule will be version bumped once dev branch is merged to master

### CODE REVIEW

File: public/media/logo/.DS_Store
Comment (Pe5h4) [inline public/media/logo/.DS_Store:1]: @petrKavulok Please, dont commit any .DS_Store files

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@petrKavulok Dont forget to update asab-webui submdule, also provide some pics how does it look like. Thx

**Comment by petrKavulok** [comment]
Screenshot 2023-05-11 at 15 30 39
Screenshot 2023-05-11 at 15 30 50
Screenshot 2023-05-11 at 15 31 24

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@petrKavulok Update changelog and submodule with the theme logo changes

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**